### DISCUSSION TITLE
Problem decoding Base64 string

### DISCUSSION THREAD

**Discussion by cranicaudal** [nextgenhealthcare/connect#6192] [Q&A]
Hi everybody,

i hope that someone could help me.

I do have a Channel getting base 64 PDF via TCP/IP. Now the Problem ist, that if the Base 64 String is decoded and the File is written, Acrobat says that the File ist corrupted.

If i put the Message directly in to Channel and do the decode everything ist ok.
I compared both Base64 strings (TCP and directly) they are exactly the same.

I have no Idea how to fix it.

**Comment by pacmano1** [comment]
Post your channel export. 

And a sample message. 

Also share how you're sending the sample message, preferably as a second mirth channel.
